src/SmartComponents/DriftPage/DriftTable/ComparisonHeader/ComparisonHeader.js

```diff
@@ -239,7 +261,9 @@ ComparisonHeader.propTypes = {
     setHistory: PropTypes.func,
     selectedHSPIds: PropTypes.array,
     selectHistoricProfiles: PropTypes.func,
-    selectedBaselineIds: PropTypes.array
+    selectedBaselineIds: PropTypes.array,
+    columnWidth: PropTypes.number,
+    setColumnHeaderWidth: PropTypes.func
```

Comment:
Since you are calling the function at the line 27, the function is supposed to be required by the component? If so, could you please add `.isRequired` to the prop type?